playbooks/kube-init.yml

```diff
@@ -0,0 +1,77 @@
+---
+- import_playbook: ka-init/init.yml
+- import_playbook: ka-lab-ipv6/ipv6-lab.yml
+  when: ipv6_enabled
+
+- hosts: master,nodes
+  become: true
+  become_user: root
+  tasks:
+    - name: Set bridge-nf-call-iptables to 1 (yes, this is redundant, earlier steps may not have been sticky)
+      shell: >
+        echo "1" > /proc/sys/net/bridge/bridge-nf-call-iptables
+      ignore_errors: true
+
+- hosts: master
+  become: true
+  become_user: root
+  # pre_tasks:
```

Comment:
Delete commented lines.

You don't need `pre_tasks` any more because tasks can now `include_role` to allow you to run roles when you want rather than always first. You can also leverage things like `include_playbook` and `tasks_from`